config/routes.rb

```diff
@@ -327,6 +327,7 @@ def external_redirect(path_prefix, target)
         end
 
         get "/editions/:id" => "editions#show"
+        get "/editions-locked/:id" => "editions#show_locked", as: :show_locked_edition
```

Comment:
I think this would be more consistent to be flagged as a member as per: https://github.com/alphagov/whitehall/blob/301072565a9dcdcf2d55c81e664497afc3654880/config/routes.rb#L311

If you defined this as `get :show_locked` you'd then get a route method of `show_locked_admin_edition_path` defined for free.

Reply:
Thanks for this - I found the routes a bit of a minefield 😱